src/win32/wazuh-installer.wxs

```diff
@@ -441,6 +441,9 @@
                     <Component Id="REPLACECONF.VBS" DiskId="1" Guid="D2CA93FD-4B3C-4147-B7D8-503487E23D16">
                         <File Id="REPLACECONF.VBS" Name="ReplaceConf.vbs" Source="ReplaceConf.vbs" />
                     </Component>
+                    <Component Id="LIBFIMDB_DLL" DiskId="1" Guid="6A4C5784-043B-4113-8688-083364757A39">

```

Comment:
Did we finally add a DLL? If not, let's remove this component until it is included.

Reply:
Done, was not necessary at the moment